docs/4.x/tagmanager/debugging.md

```diff
@@ -12,7 +12,7 @@ The debug mode lets you see for example which triggers and tags have been fired,
 To activate this mode, simply follow these steps:
 
 * Open the container in Matomo's Tag Manager and click on "Preview / Debug" in the top toolbar.
-* Open the page where you want to debug the container
+* Enter the URL or domain you want to start debugging
```

Comment:
Let's merge when we also merge the tag manager change. Then we can mention that this specific step requires Matomo 4.5 or 4.6 (depending where the feature is included)

Reply:
Yes :+1:

Reply:
@tsteur mentioned 4.6, should we merge it ?